UI/src/model/TrafficSignsCollection.js

```diff
@@ -65,7 +65,8 @@
         .then(function(assets) {
           eventbus.trigger('pointAssets:fetched');
           me.allowComplementaryIsActive(allowComplementary);
-            return filterTrafficSigns(me.filterComplementaries(assets));
+          verificationCollection.fetch(boundingBox, 300);
+          return filterTrafficSigns(me.filterComplementaries(assets));
```

Comment:
Probably we can pass to the collection the spec that has the layerName and the allowComplementary in order to have the type id. (Just to not put the 300 ). Like is being done on pointassetCollection